src/Utilities/PaymentUtils.res

```diff
@@ -1,5 +1,7 @@
+let paymentMethodListValue = Recoil.atom("paymentMethodListValue", PaymentMethodsRecord.defaultList)
```

Comment:
Please do not create any Recoil other than in RecoilAtoms. Also, instead can we keep only one source of truth for paymentMethodList and create a function which will return the extract the value from Loaded state and return instead of maintaining two separate recoils